pkg/manager/member/pd_member_manager_test.go

```diff
@@ -756,7 +769,7 @@ func newFakePDMemberManager() (*pdMemberManager, cache.Indexer, cache.Indexer) {
 	return pdManager, podIndexer, pvcIndexer
 }
 
-func newTidbClusterForPD() *v1alpha1.TidbCluster {
+func newTidbClusterForPD(tls bool) *v1alpha1.TidbCluster {
```

Comment:
What about define a new function:
```
func newTLSTidbClusterForPD() *v1alpha1.TidbCluster {
tc:=newTidbClusterForPD()
tc.Spec.TLSCluster.Enabled=true
```

Then no need to update the current files that call `newTidbClusterForPD`.